consensus/src/runway/mod.rs

```diff
@@ -238,23 +293,25 @@ where
                     self.on_parents_response(u_hash, parents)
                 }
                 Response::NewestUnit(response) => {
-                    let salt = response.as_signable().salt;
-                    trace!(target: "AlephBFT-runway", "{:?} Response parents received {:?}.", self.index(), salt);
+                    trace!(target: "AlephBFT-runway", "{:?} Response newest unit received from {:?}.", self.index(), response.index());
```

Comment:
The next PR will print errors on mismatched salts, so I don't think this will add much value then. Do you think I should still add this, or is this promise assurance enough?